roles/privacyidea/tasks/configure_privacyidea.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Copy uWSGI configuration
+  template:
+    src: privacyidea.ini.j2
+    dest: /etc/uwsgi/apps-available/privacyidea.ini
+  notify: restart uWSGI
+
+- name: Enable uWSGI app for PrivacyIDEA
+  file:
+    src: /etc/uwsgi/apps-available/privacyidea.ini
+    dest: /etc/uwsgi/apps-enabled/privacyidea.ini
+    state: link
+  notify: restart uWSGI
+
+- name: Copy uWSGI service file
+  template:
+    src: uwsgi.service.j2
+    dest: /etc/systemd/system/uwsgi.service
+  notify: restart uWSGI
+
+
+- name: Copy PrivacyIDEA WSGI script
+  template:
+    src: privacyideaapp.py.j2
+    dest: /etc/privacyidea/privacyideaapp.py
+  notify: restart uWSGI
```

Comment:
is a restart for uWSGI really necessary after each task or is it enough if it gets restarted once at the end?

Reply:
WSGI script initializes privacyidea app , any updates to the WSGI script require a restart of the uWSGI service for the changes to take effect.

Reply:
I think the "notify" only activates the handler. It is then executed only once at the end of the play.

Reply:
thank you for the explanation @YannickEvers, then everything ist fine 👍